src/sentry/static/sentry/app/views/integrationPipeline/awsLambdaFailureDetails.tsx

```diff
@@ -1,31 +1,90 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-import Button from 'app/components/button';
+import {Panel} from 'app/components/panels';
+import {IconCheckmark, IconWarning} from 'app/icons';
 import {t} from 'app/locale';
 
+import FooterWithButtons from './components/footerWithButtons';
+import HeaderWithHelp from './components/headerWithHelp';
+
+type ErrorDetail = {name: string; error: string};
+
 type Props = {
-  lambdaFunctionFailures: Array<{FunctionName: string; Runtime: string}>;
+  lambdaFunctionFailures: ErrorDetail[];
+  successCount: number;
 };
 
-export default function AwsLambdaFailureDetails({lambdaFunctionFailures}: Props) {
+export default function AwsLambdaFailureDetails({
+  lambdaFunctionFailures,
+  successCount,
+}: Props) {
+  return (
+    <React.Fragment>
+      <HeaderWithHelp docsUrl="https://docs.sentry.io/product/integrations/aws_lambda/" />
+      <Wrapper>
+        <ItemWrapper>
+          <StyledCheckmark isCircled />
+          <h3>{t('Succesfully updated %s functions', successCount)}</h3>
+        </ItemWrapper>
+        <ItemWrapper>
+          <StyledWarning />
+          <h3>{t('Failed to update %s functions', lambdaFunctionFailures.length)}</h3>
+        </ItemWrapper>
+        <StyledPanel>{lambdaFunctionFailures.map(SingleFailure)}</StyledPanel>
+      </Wrapper>
+      <FooterWithButtons buttonText={t('Finish Setup')} href="?finish_pipeline=1" />
+    </React.Fragment>
+  );
+}
+
+function SingleFailure(errorDetail: ErrorDetail) {
   return (
-    <Wrapper>
-      <h3>{t('Failed to update the following Lambda Functions')}</h3>
-      {lambdaFunctionFailures.map(func => {
-        return <div key={func.FunctionName}>{func.FunctionName}</div>;
-      })}
-      <StyledButton priority="primary" href="?finish_pipeline=1">
-        Finish
-      </StyledButton>
-    </Wrapper>
+    <StyledRow>
+      <span>{errorDetail.name}</span>
+      <Error>{errorDetail.error}</Error>
+    </StyledRow>
   );
 }
 
 const Wrapper = styled('div')`
-  margin: 20px;
+  margin: 100px 50px 50px 50px;
+`;
+
+const StyledRow = styled('div')`
```

Comment:
Any reason you couldn't use `PanelItem` here?

Reply:
@EvanPurkhiser nope, fixed